### PR TITLE
PHP 8 and Symfony 5 Support

### PR DESCRIPTION
**Pull request by BlNARYFOUR**
Added Support for PHP 8 and added Symfony 5 support (replaced deprecations).
Suggesting this to be v3.0.0

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,19 +16,22 @@
...
-        "symfony/templating": "^2.8 || ^3.0 || ^4.0",
-        "symfony/validator": "^2.8 || ^3.0 || ^4.0",
-        "symfony/yaml": "^2.8 || ^3.0 || ^4.0",
-        "psr/log": "~1.0"
+        "php": "^7.2 || ^8.0",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/intl": "^4.3 || ^5.0",
+        "symfony/templating": "^4.3 || ^5.0",
+        "symfony/validator": "^4.3 || ^5.0",
+        "symfony/yaml": "^4.3 || ^5.0",
```
Comment (hacfi) [inline composer.json:24]: Can you please allow Symfony 6 as well? Or is there additional work necessary for it to work with Symfony 6?

File: Form/Extension/ChoiceList/LocaleChoiceList.php [hunk 2]
```diff
@@ -39,7 +40,11 @@ public function __construct(LocaleInformation $information, $languagesOnly = tru
 
         foreach ($allowedLocales as $locale) {
             if ($languagesOnly && strlen($locale) == 2 || !$languagesOnly) {
-                $this->localeChoices[$locale] = Intl::getLanguageBundle()->getLanguageName($locale, $locale);
+                try {
+                    $this->localeChoices[$locale] = Languages::getName($locale, $locale);
+                } catch(MissingResourceException) {
```
Comment (hacfi) [inline Form/Extension/ChoiceList/LocaleChoiceList.php]: missing space between `catch` and `(`

**Comment by hacfi** [comment]
@BlNARYFOUR Thanks for taking the time to work on this - looks very promising!

File: composer.json (hunk 1 above)
Comment (BlNARYFOUR) [reply composer.json:24]: I think upgrading to symfony 6 would be quite easy, but didn't have the time for this yet. Maybe also better to do this in a separate MR ;)

File: Form/Extension/ChoiceList/LocaleChoiceList.php (hunk 2 above)
Comment (BlNARYFOUR) [reply Form/Extension/ChoiceList/LocaleChoiceList.php]: Fixed.

**Comment by BlNARYFOUR** [comment]
> @BlNARYFOUR Thanks for taking the time to work on this - looks very promising!

Thanks @hacfi !
I can say with confidence that I am using my fork in PHP 8 and Symfony 5 in my own project!

**Comment by BlNARYFOUR** [comment]
> @BlNARYFOUR Thanks for taking the time to work on this - looks very promising!

@hacfi Any idea if this will actually be merged in the future?

**Comment by goetas** [comment]
see https://github.com/lunetics-org/LocaleBundle/pull/218

**Pull request closed without merging**